tests/lua/match_consumed.lua

```diff
@@ -3,7 +3,7 @@ do
     return function(x)
       local a = f(1)
       if x.__tag == "None" then
-        return f(a)
+        return f(a + 0)
```

Comment:
Uggggghr. It's fine, we can discuss how to handle this later.

Reply:
won't do